Sprint-1/exercises/random.js

```diff
@@ -1,7 +1,18 @@
 const minimum = 1;
 const maximum = 100;
 
-const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
+const num = Math.floor(Math.random() * (maximum - minimum + 1))+minimum;
+
+//num1=Math.random()
+
+//console.log(num1)
+console.log(num)
+
+
+// Step 1: The built function Math.random() returns a floating point number in the range [0, 1), i.e. 0 is a possible outcome, but 1 is not inclusive.
+// Step 2: (Math.random() * (maximum - minimum + 1)) will evaluate to a floating point number between the minimum number and maximum number inclusive because of the plus 1.
```

Comment:
What would be the range `[?, ?)` of this value? 

If we were to generate a random integers between -50 to 50, will this code still work if we simply changed `minimum` and `maximum` to -50 and 50 respectively?

Reply:
> What would be the range `[?, ?)` of this value?
> 
> If we were to generate a random integers between -50 to 50, will this code still work if we simply changed `minimum` and `maximum` to -50 and 50 respectively?

The range will be (-50,50). Yes, the code will work to generate random integers between -50 and 50